apps/web/components/Charts/MixedChart/MixedChart.tsx

```diff
@@ -28,60 +34,90 @@ export const MixedChart = ({ graphData }: GraphProps) => {
   const parsedDatakeys = JSON.parse(datakeys)[0]
   const stackIds = parsedDatakeys.bars.map((e) => e.stackId)
   const shouldStack = new Set(stackIds).size !== stackIds.length
-
+  const rightPadding = parsedDatakeys.yAxis?.right ? 70 : 0
   return (
-    <ResponsiveContainer width="100%" height="100%">
-      <ComposedChart
-        width={250}
-        height={150}
-        data={parsedData}
-        margin={{
-          top: 30,
-          right: 0,
-          left: 20,
-          bottom: 5,
-        }}
+    <Box width="full" height="full">
+      <Box
+        display="flex"
+        flexDirection="row"
+        justifyContent="spaceBetween"
+        style={{ paddingRight: rightPadding }}
       >
-        <CartesianGrid strokeDasharray="1" vertical={false} color="#CCDFFF" />
-        <XAxis
-          dataKey={parsedDatakeys.xAxis}
-          stroke="#CCDFFF"
-          tick={<CustomizedAxisTick />}
-          padding={{ left: 30 }}
-          tickLine={false}
-        />
-        <YAxis yAxisId="left" stroke="#CCDFFF" tick={<CustomizedAxisTick />} />
-        <YAxis yAxisId="right" hide />
-        <Tooltip />
-        <Legend iconType="circle" align="right" content={RenderLegend} />
-        {parsedDatakeys.bars.map((item, index) => (
-          <Bar
-            key={index}
-            dataKey={item.datakey}
-            fill={item.color}
-            stackId={item.stackId}
-            barSize={16}
-            yAxisId="left"
-            radius={
-              index + 1 === parsedDatakeys.bars.length || !shouldStack
-                ? [20, 20, 0, 0]
-                : 0
-            }
-          />
-        ))}
-        {parsedDatakeys.yAxis?.left &&
-          parsedDatakeys.lines.map((item, index) => (
-            <Line
-              key={item.datakey}
-              dataKey={item.datakey}
-              stroke={item.color ? item.color : COLORS[index % COLORS.length]}
+        {parsedDatakeys.yAxis?.label && (
+          <Text variant="eyebrow">{parsedDatakeys.yAxis.label}</Text>
+        )}
+        {parsedDatakeys.yAxis?.labelRight && (
+          <Text variant="eyebrow">{parsedDatakeys.yAxis.labelRight}</Text>
+        )}
+      </Box>
+      <Box style={{ width: '100%', height: '90%' }}>
```

Comment:
It would be really nice if this was its own component (private, in the same file!).. reducing the mental overhead of parsing and understanding this whole thing